container_files/public_html/doc/builtin/procedures/Classifier.md

```diff
@@ -98,6 +98,30 @@ when a generic classification step is required.
 
 ![](%%jmlclassifier decision_tree)
 
+#### Parameter update_alg
+The parameter update_alg can take three different values: normal, gentle and prob.
+Here is how they work. We'll work with a leaf node that contains 8 positives
+and 2 negatives labels.
+
+##### prob
+It is the proportion of positive class, so simply #pos/(#pos + #neg).
```

Comment:
consider using latex formatting for formulas. you can look at the tf/idf function for an example of how to do it